test/built-ins/String/prototype/toLocaleLowerCase/special_casing.js

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2015 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Check if String.prototype.toLocaleLowerCase supports mappings defined in SpecialCasings
+info: >
+    The result must be derived according to the locale-insensitive case mappings in the Unicode Character
+    Database (this explicitly includes not only the UnicodeData.txt file, but also all locale-insensitive
+    mappings in the SpecialCasings.txt file that accompanies it).
+es5id: 15.5.4.16
+es6id: 21.1.3.22
```

Comment:
The IDs for the `toLocale...` versions of the functions are the same here as the non-locale versions. Could you update them to match the spec?